README.md

```diff
@@ -1,5 +1,7 @@
 # keen-dataviz.js
 
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fkeen%2Fkeen-dataviz.js.svg?type=shield)](https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fkeen%2Fkeen-dataviz.js?ref=badge_shield)
```

Comment:
I'm apprehensive about introducing a status badge that leads to a sign-in wall. Can you please catch me up on FOSSA and why we should include it?